src/Microsoft.AspNetCore.Http.Abstractions/HttpResponse.cs

```diff
@@ -103,5 +106,95 @@ public abstract class HttpResponse
         /// <param name="location">The URL to redirect the client to.</param>
         /// <param name="permanent"><c>True</c> if the redirect is permanent (301), otherwise <c>false</c> (302).</param>
         public abstract void Redirect(string location, bool permanent);
+
+        /// <summary>
+        /// Sends the given file using the SendFile extension.
+        /// </summary>
+        /// <param name="file">The file to send.</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public virtual Task SendFileAsync(IFileInfo file, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
These should be abstract and the implementation should be in DefaultHttpResponse like most of the other APIs. (if these should even be here. we'll discuss that separately.)